src/routes/auth.js

```diff
@@ -70,19 +75,14 @@ router.post(
 
 router.post(
 	'/create-user',
-	isAuthenticated(),
-	isAdmin,
 	validateSchema(authSchema),
+	verifyCreateToken,
 	catchAsync(async (req, res) => {
 		const {
 			firstName,
 			lastName,
 			email,
-			password,
-			gender,
-			jobRole,
-			department,
-			address
+			password
```

Comment:
@itzgeebee so we have a `/docs` endpoint which we are using to help frontend know the parameters the frontend should send

its provided [here](https://github.com/tbf-challenge/teamwork-be/blob/main/docs/swagger.yaml) and this is the production version: https://team-worker.herokuapp.com/docs

 kindly do check it out and update accordingly

Reply:
@jocrah So in the technical consideration number 2, you wrote: "We only want to take in these four fields from the body: firstName, lastName, password and email"  So that was what I followed or maybe I didn't quite understand what you meant by that. will appreciate it if you throw more light. Thanks